### PR TITLE
Rewire Item Properties to use the store

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
- Add stand-alone side drawer
- Connect the side drawer to the store
- Add the item panel to the side drawer
- Add custom highlight color to the theme

### CODE REVIEW

File: src/Components/SampleModelsControl.jsx [hunk 1]
```diff
@@ -134,13 +134,13 @@ const useStyles = makeStyles({
   root: {
     'width': '260px',
     '& .MuiOutlinedInput-input': {
-      color: 'green',
-    },
-    '& .MuiInputLabel-root': {
-      color: 'green',
+      color: 'blue',
     },
+    // '& .MuiInputLabel-root': {
```
Comment (OlegMoshkovich) [inline src/Components/SampleModelsControl.jsx:139]: @pablo-mayrgundter temporarily commenting it out - to be picked up by the theme.

File: src/Components/SampleModelsControl.jsx [hunk 2]
```diff
@@ -152,13 +152,13 @@ const useStyles = makeStyles({
       borderColor: 'gray',
     },
     '& .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-input': {
-      color: 'green',
-    },
-    '& .MuiInputLabel-root.Mui-focused': {
-      color: 'green',
+      color: 'blue',
     },
+    // '& .MuiInputLabel-root.Mui-focused': {
```
Comment (OlegMoshkovich) [inline src/Components/SampleModelsControl.jsx:157]: same

Comment (pablo-mayrgundter) [reply src/Components/SampleModelsControl.jsx:157]: For both, please convert to:
```
TODO(oleg): ... comment ...
```
so we can find later

File: src/Components/SideDrawerPanels.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import {makeStyles, useTheme} from '@mui/styles'
+import ItemProperties from './ItemProperties'
+import CloseIcon from '../assets/2D_Icons/Close.svg'
+import {TooltipIconButton} from './Buttons'
+import useStore from '../store/useStore'
+
+/**
+ * Panel Title
+ * @return {String} Panel title
```
Comment (pablo-mayrgundter) [inline src/Components/SideDrawerPanels.jsx:10]: @return -> @param for first 2

**Review by pablo-mayrgundter — APPROVED** [review]
Just a couple small issues

**Pull request merged**